content/src/content/jcr_root/apps/core/wcm/components/image/v2/image/_cq_dialog/.content.xml

```diff
@@ -151,7 +151,6 @@
                                                 name="./linkURL"
                                                 rootPath="/content"
                                                 wrapperClass="cmp-image__editor-link"/>
-                                        </items>
```

Comment:
This needs to be kept, otherwise you end up with invalid xml

Reply:
Reverted this file as we are not adding a custom field for image name at the component level.